lua/wikis/valorant/MatchPage.lua

```diff
@@ -142,7 +140,27 @@ function MatchPage:_renderGameOverview(game)
 		}
 	end
 
-	return Div{
+	local function createScoreHolder()
+		return Div{
+			classes = {'match-bm-lol-game-summary-score-holder'},
+			children = game.finished and WidgetUtil.collect(
+				not self:isBestOfOne() and Div{
```

Comment:
Move the logic out one layer instead

Reply:
What I mean is letting the "overview" part, instead of this fucntion, determine whether a scoreholder should be made. This allows for clearer separation of logic